### PR TITLE
[Breaking] Update dependencies

### PR DESCRIPTION
**Pull request by benogle**
This upgrades all dependencies in all `AnvilEmbedFrame` and `AnviSignatureModal` plus

* `AnviSignatureModal`
  * Upgrades  to use `AnvilEmbedFrame`
* `AnvilEmbedFrame`
  * Removes `enableDefaultStyles`
  * Removes `title` attribute 
* Updates all docs
* Adds an upgrading section to the readmes

Changes will result in a major version bump (v2) fro both components

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: packages/anvil-embed-frame/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@anvilco/anvil-embed-frame",
-  "version": "1.1.0",
+  "version": "2.0.0-alpha.0",
```
Comment (fostimus) [inline packages/anvil-embed-frame/package.json:3]: good to move off the alpha version?

File: packages/react-signature-modal/README.md [hunk 2]
```diff
@@ -32,24 +32,82 @@ import AnvilSignatureModal from '@anvilco/react-signature-modal'
...
+/>
+```
+
+## Upgrading from v1 to v2
+
+As of v2.0, the `AnvilSignatureModal` now uses `AnvilEmbedFrame` under the hood. The props have changed slightly:
+
+* `signURL` -> `iframeURL`
+* `onFinishSigning` -> `onEvent`. Check the `eventObject.action === 'signerComplete'` to detect a signer has finished signing
+* `onError` -> `onEvent`. Check the `eventObject.action` to determine if there is an error
```
Comment (fostimus) [inline packages/react-signature-modal/README.md:66]: should be `eventObject.error`

File: packages/react-signature-modal/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@anvilco/react-signature-modal",
-  "version": "1.8.4",
+  "version": "2.0.1-alpha.0",
```
Comment (fostimus) [inline packages/react-signature-modal/package.json:3]: same here, alpha is ok?

**Review by fostimus — APPROVED** [review]

**Review by newhouse — APPROVED** [review]

**Comment by fostimus** [comment]
Just the few small things in comments

File: packages/anvil-embed-frame/package.json (hunk 1 above)
Comment (newhouse) [reply packages/anvil-embed-frame/package.json:3]: probably waiting for `lerna publish` to handle it after merging?

Comment (benogle) [reply packages/anvil-embed-frame/package.json:3]: I will publish both repos after merge!

**Pull request merged**